content/tutorial/02-advanced-svelte/05-bindings/06-component-bindings/app-a/src/lib/Keypad.svelte

```diff
@@ -11,15 +11,9 @@
 </script>
 
 <div class="keypad">
-	<button on:click={select(1)}>1</button>
-	<button on:click={select(2)}>2</button>
-	<button on:click={select(3)}>3</button>
-	<button on:click={select(4)}>4</button>
-	<button on:click={select(5)}>5</button>
-	<button on:click={select(6)}>6</button>
-	<button on:click={select(7)}>7</button>
-	<button on:click={select(8)}>8</button>
-	<button on:click={select(9)}>9</button>
+	{#each [1, 2, 3, 4, 5, 6, 7, 8, 9] as i}
```

Comment:
While this is a clever way to write it, the previous version is more readable to novice Svelte devs because they don't have to understand `#each` blocks while solving the exercise - therefore please revert this change.